app/javascript/sync-organization-button.vue

```diff
@@ -6,42 +6,44 @@
 </template>
 
 <script>
-  import axios from 'axios';
+/* eslint-disable prefer-arrow-callback, no-alert */
+import axios from 'axios';
 
-  export default {
-    props: ['id', 'error_msg'],
-    data() {
-      return {
-        loading: false
+export default {
+  props: ['id', 'error_msg'],
+  data() {
+    return {
+      loading: false,
+    };
+  },
+  methods: {
+    load() {
+      if (this.loading) {
+        return;
       }
+      this.loading = true;
+      axios.post(`/api/organizations/${this.id}/sync`)
+        .then(function () {
```

Comment:
acá en vez de usar `function` usaría arrow functions, así no es necesario usar los `.bind(this)`.

```
.then(() => this.checkSync())
.catch(() => alert(...));
```

Reply:
Ah ya, toma el this por defecto? lo traté de usar como arrow function y me tiraba error con el bind, por eso no la usé, lo cambio.